From 9fb7196d0b6229988990a16d0697d0788daa6e47 Mon Sep 17 00:00:00 2001
From: Jonathan Davies <jpds@protonmail.com>
Date: Fri, 9 Feb 2024 13:57:34 +0000
Subject: [PATCH] cargo: Enable LTO on release build.

---
 Cargo.toml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Cargo.toml b/Cargo.toml
index 9f420c956..0e2ef9cc9 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -110,3 +110,9 @@ wasm-bindgen-crate = { version = "0.2.58", package = "wasm-bindgen" }
 # tokio = { path = "../tokio/tokio" }
 # mio = { git = "https://github.com/tokio-rs/mio.git" }
 # h2 = { git = "https://github.com/hyperium/h2.git" }
+
+[profile.release]
+lto = true
+codegen-units = 1
+opt-level = 3
+strip = true

